README.md

```diff
@@ -4,9 +4,79 @@ Ansible role to deploy and manage [openwisp-wifi-login-pages](https://github.com
 
 Required variables:
 
-- ``wifi_login_pages_domains``: a list with the hostname where the app will be reachable.
+- `wifi_login_pages_domains`: a list with the hostname where the app will be reachable.
 
 ## Deploying custom static content
 
 Custom static content (HTML files, PDF, etc.) can be deployed
-to ``{{ wifi_login_pages_path }}/static/``.
+to `{{ wifi_login_pages_path }}/static/`.
+
+## Deploy organizations configurations and assets
+
+For deploying organization YAML config files and their related static assets (logo, CSS, etc), proceed with the following tests:
+
+**Step 1**: Change directory to ansible playbook file.
+
+```
+cd <path_to_playbook_file >
+```
+
+**Step 2**: Create the directory `files`.
+
+```
+mkdir files
+```
+
+**Step 3**: Copy `organizations` directory with all the organizations configuration and assets inside the created `files` directory.
+
+```
+cp -r <path_to_organizations_directory> files
```

Comment:
I think here we should make it explicit that the resulting directory structure should be `files/organizations`, I don't think it's 100% clear